hooks/persistence-defectdojo/hook/src/main/java/io/securecodebox/persistence/config/EnvConfig.java

```diff
@@ -0,0 +1,79 @@
+package io.securecodebox.persistence.config;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * This class provides an API to access configuration values from environment variables
+ */
+@Slf4j
+public final class EnvConfig {
+
+  public boolean isDev() {
```

Comment:
@J12934 Do I understand right that this is kinda development mode to load a local k8s config from `~/.kube/config`?

Reply:
yes